### PR TITLE
Add ol-stac

### PR DESCRIPTION
**Pull request by silvester-pari**
This PR adds the ol-stac package for automatic rendering of STAC layers. Thanks @m-mohr for this package!

### CODE REVIEW

**Comment by RobertOrthofer** [comment]
I added a simple test for the STAC-Layer, using an URL from an [example](https://m-mohr.github.io/ol-stac/en/latest/examples/stac-item.html).
It looks like the properties are not passed to the super constructor of the `LayerGroup`, which the `StacLayer` ist extending. This functionality is needed here because we rely on the `id` to be set via our JSON structure.

File: elements/map/src/generate.ts [hunk 1]
```diff
@@ -1,10 +1,20 @@
 import * as olLayers from "ol/layer";
 import * as olSources from "ol/source";
 import * as olFormats from "ol/format";
+import STAC from "ol-stac/layer/STAC";
```
Comment (m-mohr) [inline elements/map/src/generate.ts:4]:
```suggestion
import STAC from "ol-stac";
```

File: elements/map/package.json [hunk 2]
```diff
@@ -5,7 +5,8 @@
   "dependencies": {
     "lit": "^2.7.4",
     "ol": "^7.4.0",
-    "ol-mapbox-style": "^10.6.0"
+    "ol-mapbox-style": "^10.6.0",
+    "ol-stac": "^1.0.0-beta.1"
```
Comment (m-mohr) [inline elements/map/package.json:9]:
```suggestion
    "ol-stac": "^1.0.0-beta.3"
```

File: elements/map/src/generate.ts (hunk 1 above)
Comment (RobertOrthofer) [reply elements/map/src/generate.ts:4]: This does not seem to work, while the properties work perfectly fine now via `import STAC from "ol-stac/layer/STAC";`

Comment (m-mohr) [reply elements/map/src/generate.ts:4]: Hmm, thanks. I'll check it.

**Review by RobertOrthofer — APPROVED** [review]
thanks @m-mohr for your work on the library, looks good to me!

File: elements/map/src/generate.ts (hunk 1 above)
Comment (silvester-pari) [reply elements/map/src/generate.ts:4]: I think it's this line that still creates a link to `index.js` https://github.com/m-mohr/ol-stac/blob/main/tasks/prepare-package.js#L13

**Comment by m-mohr** [comment]
Yeah, thanks, just found that spot, too. I released beta.4. Does it work for you now?

File: elements/map/src/generate.ts (hunk 1 above)
Comment (silvester-pari) [reply elements/map/src/generate.ts]: `beta4` fixed it! :tada:

**Pull request merged**